### PR TITLE
fixes proposed by yegor256

### PR DESCRIPTION
**Pull request by blackout314**
task #25 
- more compact mobile view
- zoom from 2 to 4

### CODE REVIEW

**Comment by 0crat** [comment]
@yegor256[/z](https://www.0crat.com/u/yegor256) please, pay attention to this pull request

**Comment by 0crat** [comment]
Job [#52](https://github.com/zold-io/zold.github.io/issues/52) is now in scope, role is `REV`

**Comment by 0crat** [comment]
@yegor256[/z](https://www.0crat.com/u/yegor256) everybody who has role `REV` is banned at [#52](https://github.com/zold-io/zold.github.io/issues/52); I won't be able to assign anyone automatically; consider assigning someone manually (as in [§19](http://www.zerocracy.com/policy.html#19)), or invite more people (as in [§51](http://www.zerocracy.com/policy.html#51)), or remove the job from the scope (as in [§14](http://www.zerocracy.com/policy.html#14))

**Comment by yegor256** [comment]
@blackout314 I don't think `zoom:4` will solve the problem with the map

**Comment by blackout314** [comment]
@yegor256 ok i will fix, is a map configuration issue

**Comment by blackout314** [comment]
@yegor256 no example for avoid gray border
https://developers.google.com/maps/documentation/javascript/examples/control-default

all google maps in documentation show gray borders

**Comment by yegor256** [comment]
@blackout314 see above. Also, when you copy some code from SO, it's better to place a link in the code (a comment) explaining where the code is coming from. This will help understand your intention, in the future.

**Comment by blackout314** [comment]
@yegor256 ok boss, is one of many try because isnt supported by google maps v3, is a workaround

File: html/map.html [hunk 1]
```diff
@@ -31,7 +31,7 @@
     <meta name="keywords" content="zerocracy, management, robotocracy, meritocracy"/>
     <meta name="viewport" content="width=device-width, initial-scale=1.0"/>
     <link rel="shortcut icon" href="https://www.zold.io/logo.png"/>
-    <link href="@@VERSION/css/map.min.css" rel="stylesheet"/>
+    <link href="@@VERSION/css/map.min.css?2" rel="stylesheet"/>
```
Comment (yegor256) [inline html/map.html]: @blackout314 this seems to be a typo, right?

File: js/map.js [hunk 2]
```diff
@@ -24,6 +24,15 @@ SOFTWARE.
 
 function init() {
   startLoader();
+  function handle() {
+    var proj = map.getProjection();
```
Comment (yegor256) [inline js/map.js]: @blackout314 what this variable is for?

File: js/map.js [hunk 3]
```diff
@@ -24,6 +24,15 @@ SOFTWARE.
...
+    var bounds = map.getBounds();
```
Comment (yegor256) [inline js/map.js]: @blackout314 what this is for?

**Comment by yegor256** [comment]
@blackout314 see above

File: html/map.html (hunk 1 above)
Comment (blackout314) [reply html/map.html]: @yegor256 no, is for force download new version for browser

**Comment by blackout314** [comment]
@yegor256 code cleaned

**Comment by yegor256** [comment]
@rultor merge

**Comment by 0crat** [comment]
Job `gh:zold-io/zold.github.io#52` is not assigned, can't get performer

**Comment by 0crat** [comment]
The job [#52](https://github.com/zold-io/zold.github.io/issues/52) is now out of scope

**Pull request merged**